ql/src/java/org/apache/hadoop/hive/ql/parse/TezCompiler.java

```diff
@@ -2054,7 +2067,8 @@ private void markSemiJoinForDPP(OptimizeTezProcContext procCtx)
               // Lookup nDVs on TS side.
               RuntimeValuesInfo rti = procCtx.parseContext
                       .getRsToRuntimeValuesInfoMap().get(rs);
-              ExprNodeDesc tsExpr = rti.getTsColExpr();
+              // TODO Adapt for multi column semi-joins.
```

Comment:
TODO?

Reply:
I meant to handle this as part of HIVE-23934. I added the reference to the comment.